File: app/src/main/java/com/startogamu/zikobot/Converter.java
package com.startogamu.zikobot;

import android.support.v7.widget.RecyclerView;
import android.widget.TimePicker;

import com.joxad.androidtemplate.core.view.utils.EndlessRecyclerOnScrollListener;

import java.util.ArrayList;

/**
 * Created by Jocelyn on 21/08/2017.
 */

public class Converter {

    public void init() {
        TimePicker timePicker = new TimePicker(null);
        timePicker.setOnTimeChangedListener(new TimePicker.OnTimeChangedListener() {
            @Override
            public void onTimeChanged(TimePicker timePicker, int i, int i1) {

            }
        });
    }
}

File: app/src/main/java/com/startogamu/zikobot/Converter.kt
package com.startogamu.zikobot

import android.support.v7.widget.RecyclerView
import android.view.KeyEvent
import android.view.View
import android.widget.EditText
import android.widget.TextView
import android.widget.TimePicker

import com.joxad.androidtemplate.core.view.utils.EndlessRecyclerOnScrollListener

import java.util.ArrayList

/**
 * Created by Jocelyn on 21/08/2017.
 */

class Converter {

    fun init() {
        val timePicker = TimePicker(null)
        timePicker.setOnTimeChangedListener { timePicker, i, i1 -> }
        val et = EditText(null)
        et.setOnEditorActionListener { textView, i, keyEvent -> false }
    }
}
